src/renderer/families/tron/TransactionConfirmFields.js

```diff
@@ -0,0 +1,99 @@
+// @flow
+
+import invariant from "invariant";
+import React from "react";
+import styled from "styled-components";
+import { Trans } from "react-i18next";
+
+import type { AccountLike, Account, Transaction } from "@ledgerhq/live-common/lib/types";
+import { getMainAccount } from "@ledgerhq/live-common/lib/account";
+import type { ThemedComponent } from "~/renderer/styles/StyleProvider";
+
+import TransactionConfirmField from "~/renderer/components/TransactionConfirm/TransactionConfirmField";
+import Text from "~/renderer/components/Text";
+import WarnBox from "~/renderer/components/WarnBox";
+import Box from "~/renderer/components/Box";
+
+const Info: ThemedComponent<{}> = styled(Box).attrs(() => ({
+  ff: "Inter|SemiBold",
+  color: "palette.text.shade100",
+  mt: 6,
+  mb: 4,
+  px: 5,
+}))`
+  text-align: center;
+`;
+
+const AddressText = styled(Text).attrs(() => ({
+  ml: 1,
+  ff: "Inter|Medium",
+  color: "palette.text.shade80",
+  fontSize: 3,
+}))`
+  word-break: break-all;
+  text-align: right;
+  ma-wwidth: 50%;
+`;
+
+const Pre = ({
+  account,
+  parentAccount,
+  transaction,
+}: {
+  account: AccountLike,
+  parentAccount: ?Account,
+  transaction: Transaction,
+}) => {
+  const mainAccount = getMainAccount(account, parentAccount);
+  // const explorerView = getDefaultExplorerView(mainAccount.currency);
```

Comment:
could remove if not used? was it planned to be used?

Reply:
It will be used on the vote delegation bt i can remove it for now :+1: